src/Pelagos/Bundle/AppBundle/Resources/views/Search/form.html.twig

```diff
@@ -1,6 +1,9 @@
 <form id="searchForm" name="searchForm" action="{{ path('pelagos_app_ui_searchpage_default') }}" method="get" class="search-form">
     <div id="custom-search-input">
-        <div class="col-md-2"></div>
+        <div class="col-md-2">
+            <input type="date" id="collectionStartDate" name="collectionStartDate" class="input-group form-control" value="{{ (collectionStartDate) ? collectionStartDate : null }}">
```

Comment:
These inputs could use some labels, to indicate what they are for. But it can be done later also.